src/index.ts

```diff
@@ -1,2 +1,42 @@
 #!/usr/bin/env node
-console.log('aa');
\ No newline at end of file
+import { config as loadEnv } from 'dotenv';
+import { commands, help } from './commands';
+
```

Comment:
It is ok if you use a library to handle command line arguments, no need to be hardcoding that yourself

Reply:
Yes, I'd totally do that in real life project. But I took this dynamic command builder more as a challenge for me.